### DISCUSSION TITLE
Power questions

### DISCUSSION THREAD

**Discussion by doc1011** [CircuitSetup/Time-Circuits-Display#30] [General]
Hi there - great piece of kit! 

I've got a 12v DC power adapter however I'm wondering how to connect it to the green connector at the back of the keypad board. The adapter comes with a green tip with the ability to wire something in, however it doesn't fit into the female connector on the board. It there an adapter you can recommend or am I just thick? For reference here is the adapter & included tips: https://www.amazon.co.uk/gp/product/B0BKZ9V5K2/ref=ppx_yo_dt_b_asin_title_o00_s00?ie=UTF8&psc=1. 

Also, there appears to be a 5v pinout right next to the 12v connector - is it therefore possible to hardwire in a 5v power source direct? 

Thanks!

**Comment by CircuitSetup** [comment]
Thanks for purchasing!

The control board should have had the other side of the green screw connector packaged with it. If I forgot this, my apologies - I can either send you one or get you the exact part number, so you can purchase it locally (much faster if not in the US).

For the 5v input, that is for directly providing a 5v regulated power source and bypasses the on-board regulator. That would require some soldering though. 

Alternatively, you can power everything with a 5v mini-usb adapter (plugged into the esp32 on the control board) as long as it outputs at least 1A.

**Reply by doc1011** [reply]
Thanks for the quick reply! 

You are absolutely right, the screw connector bit was indeed in the box - I missed it! Thanks so much.

Would just like to say the service and support has been top notch - I had another query via the website when I first bought the unit which was dealt with very quickly and I will gladly purchase more again in future... I'm keeping my eyes peeled for the speedo! 

Cheers!
